apps/src/lib/ui/accounts/ManageLinkedAccounts.jsx

```diff
@@ -237,24 +241,34 @@ class OauthConnection extends React.Component {
               action={oauthToggleConnectionPath}
             >
               {/* This button intentionally uses BootstrapButton to match other
-                  account page buttons */}
+                  account page buttons.
+                  This button is disabled according to disconnectDisabledStatus or
+                  when the user is attempting this action from the Maker App for
+                  their Google Account. This action is blocked due to Google authentication
+                  security protocols.
+                  */}
               <BootstrapButton
                 type="submit"
                 style={styles.button}
                 text={buttonText}
-                disabled={!!disconnectDisabledStatus}
+                disabled={buttonDisabled}
               />
               <RailsAuthenticityToken />
             </form>
-            {disconnectDisabledStatus && (
+            {buttonDisabled && (
               <ReactTooltip
                 id={tooltipId}
                 offset={styles.tooltipOffset}
                 role="tooltip"
                 effect="solid"
               >
                 <div style={styles.tooltip}>
-                  {this.getDisconnectDisabledTooltip()}
+                  {/* There are two causes for errors: disconnectDisabledStatus and logging in to
+                      Google from the Maker App. Display the appropriate error text.
+                    */}
+                  {disconnectDisabledStatus
+                    ? this.getDisconnectDisabledTooltip()
+                    : i18n.manageLinkedAccounts_makerAuthError()}
```

Comment:
i think it might be clearer to move this comment and calculation to the top of `render` in place of calculating `buttonDisabled` above. we could do something like:

```
let disconnectDisabledMessage;
if (isCodeOrgBrowser() && credentialType === OAuthProviders.google) {
  disconnectDisabledMessage = i18n.manageLinkedAccounts_makerAuthError();
} else if (!!disconnectDisabledStatus) {
  disconnectDisabledMessage = this.getDisconnectDisabledTooltip();
}
```

and use the presence of `disconnectDisabledMessage` in the same way we're using `buttonDisabled`